calico/security/comms/crypto-auth.md

```diff
@@ -113,3 +136,5 @@ For detailed reference information on these parameters, refer to:
 - **Typha**: [Felix-Typha TLS configuration](../../reference/typha/configuration#felix-typha-tls-configuration)
 
 - **Felix**: [Felix-Typha TLS configuration](../../reference/felix/configuration#felix-typha-tls-configuration)
+
```

Comment:
nit: I realize it might not be entirely necessary from a functional standpoint, but I think we should probably close off the tabs with `%>` before doing the `{% endtabs %}` tag just for neatness's sake.